lib/phlex/context.rb

```diff
@@ -80,12 +80,21 @@ def _attributes(attributes)
         end
       end
 
-      attributes.transform_values! { CGI.escape_html(_1) }
+      attributes.transform_values! do |value|
+        next value if (value == true || value == false)
+        CGI.escape_html(value)
+      end
 
       attributes[:href].sub!(/^\s*(javascript:)+/, "") if attributes[:href]
 
       attributes.each do |k, v|
-        buffer << Tag::SPACE << k.name << Tag::EQUALS_QUOTE << v << Tag::QUOTE
+        next unless v
+
+        if v == true
```

Comment:
@joeldrapper shouldn't we check for the `false` value here as well?

Reply:
It's checked on line 91.